### PR TITLE
[bn] localize cncf-code-of-conduct.md

### PR DESCRIPTION
**Pull request by sajibAdhi**
Bengali Localization of `content/bn/community/static/cncf-code-of-conduct.md`.
This is a copy from [cncf/foundation](https://github.com/cncf/foundation/blob/main/code-of-conduct-languages/bn.md)

Fixes Issue: #44993
Resolves PR: #45009 

Tracking Issue: #31677

### CODE REVIEW

File: content/bn/community/static/cncf-code-of-conduct.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
+<!-- Do not edit this file directly. Get the latest from
+     https://github.com/cncf/foundation/blob/main/code-of-conduct.md -->
```
Comment (sftim) [inline content/bn/community/static/cncf-code-of-conduct.md:2]: That's not the right link. Did you mean https://github.com/cncf/foundation/blob/main/code-of-conduct-languages/bn.md?

File: content/bn/community/static/cncf-code-of-conduct.md
Comment (sftim) [inline content/bn/community/static/cncf-code-of-conduct.md:1]: There's another file, [`content/bn/community/code-of-conduct.md`](https://github.com/kubernetes/website/blob/62ede117d7d9c3a60c9609cdb1154cc35aae3275/content/bn/community/code-of-conduct.md), that needs to change at the same time.

Comment (asem-hamid) [reply content/bn/community/static/cncf-code-of-conduct.md:1]: @sftim, I've already updated that in this [PR](https://github.com/kubernetes/website/pull/46048/files#diff-5d36dcf2469407150c30716e5cf9285cf3ba1d7d1eb4f346941df6b56e4004df) recently. After that, I asked him to add CoC in the bn/community/static directory. Both are synchronized well.

File: content/bn/community/static/cncf-code-of-conduct.md (hunk 1 above)
Comment (sajibAdhi) [reply content/bn/community/static/cncf-code-of-conduct.md:2]: Yes.

**Comment by asem-hamid** [comment]
/lgtm now
Thank you @sftim and @sajibAdhi

**Review by mitul3737 — APPROVED** [review]

**Review by Imtiaz1234 — APPROVED** [review]

**Pull request merged**